lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -65,7 +65,12 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL, $fail
   // aware of a user's northstar id. If it doesn't find one, we just grab it
   // from northstar directly.
   if (!$northstar_id) {
-    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+    if ($user->uid) {
+      $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+    } else {
+      $northstar_user = dosomething_northstar_get_user($values['northstar_id']);
+      $user = user_load($northstar_user->drupal_id);
```

Comment:
I think that's why we were getting the below error when the cron job ran too because there was no `$user->uid` (or `$user`) in line 69.
```
WD cron: DoSomething\Gateway\Exceptions\InternalException: Exception     [error]
in Northstar "GET v1/users/drupal_id/" endpoint: [405] Client error:
`GET https://profile.dosomething.org/v1/users/drupal_id/` resulted in
a `405 Method Not Allowed` response:
{"error":{"code":405,"message":""}}
 in DoSomething\Gateway\Common\RestApiClient->send() (line 239 of
/var/www/www.dosomething.org/releases/20170103183811/vendor/dosomething/gateway/src/Common/RestApiClient.php).
```